src/components/screens/ExperimentView.vue

```diff
@@ -178,36 +178,35 @@
 
 <script>
 
+import axios from 'axios'
 import _ from 'lodash'
 import {marked} from 'marked'
 import config from '@/config'
 import Button from 'primevue/button'
 import Chip from 'primevue/chip'
+
 import DefaultLayout from '@/components/layout/DefaultLayout'
 import PageLoading from '@/components/common/PageLoading'
 import ItemNotFound from '@/components/common/ItemNotFound'
 import ProgressSpinner from 'primevue/progressspinner'
+import useAuth from '@/composition/auth'
 import useItem from '@/composition/item'
 import useFormatters from '@/composition/formatters'
-import axios from 'axios'
+import config from '@/config'
 import { oidc } from '@/lib/auth'
```

Comment:
This `import config from '@/config'` isn't needed here (duplicated from line 184), and I believe the `import { oidc } from '@/lib/auth'` references a method removed by these changes. Both of these throw compiler errors when running `npm run dev`.

Reply:
Thanks, not sure how that got into the checked-in version. I've removed it.